apm-commons/apm-datacarrier/src/test/java/org/apache/skywalking/apm/commons/datacarrier/common/AtomicRangeIntegerTest.java

```diff
@@ -39,4 +48,95 @@ public void testGetAndIncrement() {
         Assert.assertEquals(1, (int)atomicI.floatValue());
         Assert.assertEquals(1, (int)atomicI.doubleValue());
     }
+
+    @Test
+    @Benchmark
+    public void testOriGetAndIncrementPerformance() {
+        ATOMIC_ORI.oriGetAndIncrement();
+    }
+
+    @Test
+    @Benchmark
+    public void testNewGetAndIncrementPerformance() {
+        ATOMIC_NEW.getAndIncrement();
+    }
+
+    public static void main(String[] args) throws RunnerException {
+        Options opt = new OptionsBuilder()
+                .include(AtomicRangeIntegerTest.class.getSimpleName())
+                .forks(1)
+                .threads(4)
+                .output("/tmp/jmh.log")
+                .warmupIterations(0)
```

Comment:
Add warmup to make the result more stable, at least 3.

Reply:
I have added, and add syncIterations false.